packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html

```diff
@@ -0,0 +1,70 @@
+<div class="p-5">
+    <oui-back-button
+        data-on-click="$ctrl.goBack()">{{:: 'pci_projects_project_storages_containers_add_back_label' | translate }}</oui-back-button>
```

Comment:
Prefer using `ngBind` over `{{`.

Reply:
> We said we prefer ngBind over {{ because if you don't put :: before your translate filter, angular will digest all translations for each $digest :/

One time binding in this case, no worries about `{{` :)